doc/build/build_options.md

```diff
@@ -157,6 +157,9 @@ independently of oneDNN.
 For a debug build of oneDNN it is advisable to specify a Compute Library build
 which has also been built with debug enabled.
 
+@warning
+oneDNN is only compatible with Compute Library builds v20.11 or later.
```

Comment:
Could you update the CMAKE and build from source instructions to reflect this?
(see this [PR](https://github.com/oneapi-src/oneDNN/pull/905/files))

tagging @nSircombe @diaena for further comments

Reply:
Good point @mgouicem - the [build from source instructions](https://github.com/oneapi-src/oneDNN/blob/a18f78f1f058437e9efee403655d671633360f98/doc/build/build.md) should be updated too.

As it stands with this patch, v20.11 will be required since `NEGEMMConv2D` was introduced with the 20.11 release. 20.11 doesn't require c++14, but the current master, and imminent v21.02 release, do. Once 21.02 is released, it will be worth returning to the `CMAKE_CXX_STANDARD 14` / `CMAKE_CXX_EXTENSIONS` issue and consider incrementing the recommended ACL version to 21.02 and adding in the extra CMake flags to `ACL.cmake`.

Reply:
Thanks for the clarification. I guess the bigger point (other than the C++14 part) is:
- if we plan to change the minimum supported version of ACL to v20.11+, we should check that at build time and document it.
- if we plan to continue to support versions of ACL older than v20.11, then we should enable this implementation only when the ACL version requirement is met, and guard calls to new functions (e.g. with macro guards in the code, or in CMAKE during build).

Reply:
Good afternoon, @mgouicem.
> if we plan to continue to support versions of ACL older than v20.11

There seems to be no such goal, the idea is to set the minimum supported version. One of the reasons is that the version v20.08 (and older ones) contain bugs, which were fixed in v20.11.

Reply:
Looks good to me! Thanks for the clarification and code update!